egs2/meld/asr1/local/path.sh

```diff
@@ -0,0 +1,4 @@
+if ! which ffmpeg &> /dev/null; then 
```

Comment:
I remember the ffmpeg is automatically installed during the time of torchaudio installation? Do we need this check

Reply:
This check helped me discover `ffmpeg` was missing in my environment as I re-run this recipe. Found something [here](https://pytorch.org/audio/master/tutorials/streamwriter_basic_tutorial.html#:~:text=TorchAudio%20dynamically%20loads%20compatible%20FFmpeg%20libraries%20installed%20on%20the%20system.), seems that torchaudio will automatically load ffmpeg, given ffmpeg is installed. If that's the case, we may need this check.

Reply:
I see. I got it installed when using conda, but maynot with pip. So we can just keep it~